recurring_contract/models/contract_group.py

```diff
@@ -80,17 +77,10 @@ def write(self, vals):
             Perform various check at contract modifications
             - Advance billing increased or decrease
             - Recurring value or unit changes
-            - Another change method was selected
         """
         res = True
-        for group in self:
-            # Get the method to apply changes
-            change_method = vals.get('change_method', group.change_method)
-            change_method = getattr(group, change_method)
-
-            res = super(ContractGroup, group).write(vals) & res
-            change_method()
-
+        res = super(ContractGroup, self).write(vals) & res
```

Comment:
`res` is always True, it can be removed from line 81 as well.
```suggestion
        res = super().write(vals)
```